code/_core/obj/structure/interactive/drop_pod.dm

```diff
@@ -154,7 +153,9 @@ var/global/list/turf/drop_pod_turfs = list() //Drop pods that need to respawn.
 			for(var/k in contents)
 				var/atom/movable/M = k
 				M.force_move(src.loc)
-
+			CALLBACK("set_state_\ref[src]",50,src,.proc/set_state,caller,POD_DESTROY,desired_loc)
+		if(POD_DESTROY)
+			src.Destroy()
```

Comment:
qdel should be used here instead of src.Destroy()

Reply:
Wouldnt calling Qdel leave it in the subsystem for all drop pods? Calling destroy here makes sure to remove it from that subsystem by properly running its destroy code.

Reply:
Nope. qdel runs src.Destroy properly. qdel is basically calling Destroy but with some extra checks